packages/faustwp-core/package.json

```diff
@@ -20,15 +20,16 @@
     "@types/jest": "^27.0.2",
     "@types/lodash": "^4.14.176",
     "@types/node": "^17.0.17",
+    "fetch-mock": "9.11.0",
     "jest": "^27.3.1",
     "rimraf": "^3.0.2",
     "ts-jest": "^27.0.7",
     "ts-loader": "^9.2.6",
-    "typescript": "^4.4.4",
-    "fetch-mock": "9.11.0"
+    "typescript": "^4.4.4"
   },
   "dependencies": {
     "@wordpress/hooks": "^3.14.0",
+    "chalk": "^4.1.2",
```

Comment:
Using Chalk >5 because it is ESM only.

Reply:
Also, this dependency won't effect bundle size much since @faustwp/core requires @faustwp/cli, and @faustwp/cli has a dependency of chalk, so it will be deduped.